low-level/framework/rabbitmq/rabbitmq_ingress_processor.py

```diff
@@ -228,62 +221,38 @@ def _process_msg(self, ch, method, properties, body):
-            self._queue_name    = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.QUEUE_NAME}",
-                                                                 'actuator-req-queue')
-            self._exchange_name = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.EXCHANGE_NAME}",
-                                                                 'sspl-in')
-            self._routing_key   = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.ROUTING_KEY}",
-                                                                 'actuator-req-key')
-            self._username = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.USER_NAME}",
-                                                                 'sspluser')
-            self._password = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.PASSWORD}",'')
-
-            cluster_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{self.CLUSTER_ID_KEY}",'CC01')
-
-            node_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{self.NODE_ID_KEY}",'SN01')
-            # Decrypt RabbitMQ Password
-            decryption_key = encryptor.gen_key(cluster_id, ServiceTypes.RABBITMQ.value)
-            self._password = encryptor.decrypt(decryption_key, self._password.encode('ascii'), "RabbitMQingressProcessor")
-
-            # Create a routing key unique to this instance
-            unique_routing_key = f'{self._routing_key}_node{node_id}'
-            logger.info(f"Connecting using routing key: {unique_routing_key}")
-            self._connection = RabbitMQSafeConnection(
-                self._username, self._password, self._virtual_host,
-                self._exchange_name, unique_routing_key, self._queue_name
-            )
-        except Exception as ex:
-            logger.error("RabbitMQingressProcessor, _configure_exchange: %r" % ex)
+        self._node_id = Conf.get(SSPL_CONF,
+                                 f"{CLUSTER}>{SRVNODE}>{self.NODE_ID_KEY}",
+                                 'SN01')
+        self._consumer_id = Conf.get(SSPL_CONF,
+                                     f"{self.RABBITMQPROCESSOR}>{self.CONSUMER_ID}",
+                                     'sspl_actuator')
+        self._consumer_group = Conf.get(SSPL_CONF,
+                                        f"{self.RABBITMQPROCESSOR}>{self.CONSUMER_GROUP}",
+                                        'cortx_monitor')
+        self._message_type = Conf.get(SSPL_CONF,
+                                      f"{self.RABBITMQPROCESSOR}>{self.MESSAGE_TYPE}",
+                                      'requests')
+        self._offset = Conf.get(SSPL_CONF,
+                                f"{self.RABBITMQPROCESSOR}>{self.OFFSET}",
+                                'earliest')
 
     def shutdown(self):
         """Clean up scheduler queue and gracefully shutdown thread"""
         super(RabbitMQingressProcessor, self).shutdown()
-        try:
-            self._connection.cleanup()
```

Comment:
no connection cleanup/close needed on message bus (kafka) side? Should confirm with foundation team

Reply:
I have confirmed with foundation team, its not there.
```
[3/1 4:06 PM] Sandeep Anjara
    Is there any method to close connection ?
​[3/1 4:08 PM] Selva Nambi
    No for now
```

Reply:
ok, keep a TODO still to check later again, have request Sachin to check on this further

Reply:
add a TODO plz

Reply:
Done